adapters/Vungle/VungleAdapter/GADMAdapterVungleInterstitial.m

```diff
@@ -31,7 +33,7 @@ - (instancetype)initWithGADMAdNetworkConnector:(id<GADMAdNetworkConnector>)conne
 }
 
 - (void)dealloc {
-  [self stopBeingDelegate];
+  //[self stopBeingDelegate];
```

Comment:
Just for reserving.@ashinagawa Hi Aki, do we need to remove dealloc method, it seems stopBeingDelegate will be called every time before dealloc method is called. Thanks. :)

Reply:
Yes. as long as `stopBeingDelegate` is called properly, we don't need `dealloc` anymore.